ccd-definition/CaseEvent/StateChange.json

```diff
@@ -41,7 +41,7 @@
     "Name": "Populate standard directions",
     "Description": "Populate standard directions",
     "DisplayOrder": 14,
-    "PreConditionState(s)": "Submitted",
+    "PreConditionState(s)": "*",
```

Comment:
Are we sure that we want this to end in the gatekeeping state now? If we are calling this event after the add-hearing-bookings event it might be a bit confusing